src/gui/my_controls.cpp

```diff
@@ -1003,7 +1003,7 @@ wxString RefinementPackageListControl::OnGetItemText(long item, long column) con
 
     MyRefinementPackageAssetPanel* parent_panel = reinterpret_cast<MyRefinementPackageAssetPanel*>(m_parent->GetParent( )->GetParent( )); // not very nice code!
 
-    if ( parent_panel->all_refinement_packages.GetCount( ) > 0 ) {
+    if ( parent_panel->all_refinement_packages.GetCount( ) > 0 && item < parent_panel->all_refinement_packages.GetCount()) {
```

Comment:
Nice catch! I vaguely remember running into this at least once before, but didn't find the fix.

Are the conditionals on (or around) lines 1055 and 1160 possibly needing a similar amendment?